uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Page/Dashboard.php

```diff
@@ -13,6 +13,7 @@ public function run() {
         'administer leave and absences',
         'access leave and absences in ssp',
         'manage leave and absences in ssp',
+        'can administer calendar feeds'
```

Comment:
Sorry to meddle with this, but, would this block admins from seeing the whole dashboard just because they can't administer calendar feeds?

Reply:
It won't prevent the admin inasmuch as the admin still has the required permission (`access leave and absences`) to view the page